### ISSUE TITLE
Issues with JEI

### ISSUE THREAD

**Issue by bookerthegeek** [jaredlll08/CraftTweaker#212]
### Original Post
https://github.com/BeetoGuy/BetterWithMods/issues/336

#### Issue Description:
Slight issue with JEI integration. After removing a recipe, then adding it back in, it does not show up in JEI. Still works, just does not show up. Interestingly Enough, it does show up in the Kiln recipe list...

#### What happens:
Recipes are not showing up in the correct place.

#### What you expected to happen:
Recipes to show up correctly.

#### Script used 
```
mods.betterwithmods.Saw.remove(<betterwithmods:debarked_old>);
mods.betterwithmods.Saw.add([<betterwithmods:wood_siding> * 2], <betterwithmods:debarked_old>);
```
#### Proof
![No Recipe](https://i.imgur.com/0cDRyJY.png)
![WhatTheHell](https://i.imgur.com/Hdh5ZZU.png)

#### Minetweaker.log file 
```
[AVAILABLE][CLIENT][INFO] MineTweaker: Building registry
[AVAILABLE][CLIENT][INFO] MineTweaker: Sucessfully built item registry
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding command minetweaker
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding minetweaker command handextra
[SERVER_ABOUT_TO_START][SERVER][INFO] Removing 1 saw Recipe(s) for Oak Debarked Log
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding 1 saw Recipe(s) for Oak Debarked Log
[SERVER_ABOUT_TO_START][SERVER][INFO] 
```
___
#### Affected Versions (Do *not* use "latest"):
- Minecraft: 11.2
- Forge: 1.11.2-13.20.0.2296-universal
- CraftTweaker: 1.11.2-3.0.24
- JEI: 1.11.2-4.3.5.275

#### Most recent log file where the issue was present: 
[gist link here](https://gist.github.com/bookerthegeek/f507c9e5985a949d9c74dc2401198628)

**Comment by rinart73** [comment]
I'm having the same issue. It appears randomly though.

**Comment by jaredlll08** [comment]
JEI support has been fixed for everything in 1.12 and 1.11.2 isn't supported anymore
